lib/components/Sequencer/SequencerComponents/TrackRack.jsx

```diff
@@ -43,6 +42,11 @@ const TrackRack = (props) => {
           />
         )}
       </div>
+      <div className='title-bar'>
+        <div className="sample-name">{props.name}</div>
+        <button onClick={(e)=>{props.removeTrack(props.name)}}>delete</button>
```

Comment:
@DanGrund Is this the button for removing a track from the sequencer? If so should it be "Remove" vs. "Delete"?